clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleSslLoadBalancerAtomicOperation.groovy

```diff
@@ -173,7 +173,7 @@ class DeleteGoogleSslLoadBalancerAtomicOperation extends DeleteGoogleLoadBalance
         "Health check $healthCheckName",
         project,
         task,
-        BASE_PHASE,
+        [action: 'delete', operation: 'compute.healthChecks.delete', phase: BASE_PHASE, (TAG_SCOPE): SCOPE_GLOBAL],
```

Comment:
I was tempted to do a big refactoring here but I dont want to break it. Instead I saved the fight for another day when I have more time. There are wrappers upon wrappers here and redundant parameters to go around. I have some thoughts about how to approach this
but probably not the time for a few weeks.